### PR TITLE
Import initial code

### PR DESCRIPTION
**Pull request by bessbd**
This change does the initial code import.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## PR summary
+
+<!-- please include a brief summary of the changes in this PR -->
+
+Fixes: <! -- link to issue -->
+
+**Note: An existing issue is [required](https://github.ibm.com/cloudant/cloudant-node-sdk/blob/master/CONTRIBUTING.md#PRs) before opening a PR.**
```
Comment (emlaver) [inline .github/pull_request_template.md:7]:
```suggestion
**Note: An existing issue is [required](https://github.com/IBM/cloudant-node-sdk/blob/master/CONTRIBUTING.md#PRs) before opening a PR.**
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,727 @@
...
+  The example codes and outputs below are generated using the `embedmd` go package.
+
+      https://github.com/campoy/embedmd
+
+  You should regenerate the example codes after making any changes to examples in the test/examples/ folder.
+
+      embedmd -w README.md
+  -->
+
+[![Build Status](https://travis.ibm.com/IBM/cloudant-node-sdk.svg?token=o99uRFDJzv9wzzsenSC6&branch=master)](https://travis.ibm.com/IBM/cloudant-node-sdk)
```
Comment (emlaver) [inline README.md:11]:
```suggestion
[![Build Status](https://travis-ci.com/IBM/cloudant-node-sdk.svg?token=o99uRFDJzv9wzzsenSC6&branch=master)](https://travis.ibm.com/IBM/cloudant-node-sdk)
```

File: README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
       embedmd -w README.md
   -->
 
-[![Build Status](https://travis.ibm.com/IBM/cloudant-node-sdk.svg?token=o99uRFDJzv9wzzsenSC6&branch=master)](https://travis.ibm.com/IBM/cloudant-node-sdk)
+[![Build Status](https://travis-ci.com/IBM/cloudant-node-sdk.svg?token=o99uRFDJzv9wzzsenSC6&branch=master)](https://travis.ibm.com/IBM/cloudant-node-sdk)
```
Comment (emlaver) [inline README.md:11]: We'll need to remove the token as well!
```suggestion
[![Build Status](https://travis-ci.com/IBM/cloudant-node-sdk.svg?branch=master)](https://travis.ibm.com/IBM/cloudant-node-sdk)
```

**Comment by bessbd** [comment]
Thank you for the review, @emlaver and @ricellis . I'm about to merge this.

**Pull request merged**